identity/webapp/src/frontend/MyAccount/ChangePassword.tsx

```diff
@@ -40,6 +41,10 @@ export const ChangePassword: React.FC<ChangeDetailsModalContentProps> = ({ onCom
         setError('password', { type: 'manual', message: 'Incorrect password.' });
         break;
       }
+      case RequestDeleteError.BRUTE_FORCE_BLOCKED: {
```

Comment:
As above, we're not requesting a delete here.

```suggestion
      case UpdatePasswordError.BRUTE_FORCE_BLOCKED: {
```

Reply:
Copy and paste error. Shame on me.